docs/examples/students_t_distribution.md

```diff
@@ -0,0 +1,106 @@
+
+The t-distribution is widely used in statistical analysis such as in Student's t-test, confidence interval estimation and linear regression analysis. It is often used in Bayesian analysis particularly when the sample size is small or the population variance is unknown. The t-distribution is more robust to outliers and deviations from normality compared to the normal distribution, making it a suitable choice for modeling data with heavier tails. 
+
+
+## Probability Density Function (PDF):
+
+```{code-cell}
+---
+tags: [remove-input]
+mystnb:
+  image:
+    alt: Student's t Distribution PDF
+---
+
+import matplotlib.pyplot as plt
+import arviz as az
+from preliz import StudentT
+az.style.use('arviz-doc')
+nus = [2., 5., 5.]
+mus = [0., 0.,  -4.]
+sigmas = [1., 1., 2.]
+for nu, mu, sigma in zip(nus, mus, sigmas):
+    StudentT(nu, mu, sigma).plot_pdf(support=(-10,6))
+
+plt.savefig('students_t.png')
+```
+
+## Cumulative Distribution Function (CDF):
+
+```{code-cell}
+---
+tags: [remove-input]
+mystnb:
+  image:
+    alt: Student's t Distribution CDF
+---
+
+for nu in nus:
+    StudentT(nu, mu, sigma).plot_cdf(support=(-10,6))
+```
+
+## Key properties and parameters:
+
+```{eval-rst}
+========  ==========================================
+Support   :math:`x \in \mathbb{R}`
+Mean      :math:`\mu` for :math:`\nu > 1`, otherwise undefined
+Variance  :math:`\frac{\nu}{\nu-2}` for :math:`\nu > 2`,
+          :math:`\infty` for :math:`1 < \nu \le 2`, otherwise undefined
+========  ==========================================
+```
+
+**Probability Density Function (PDF):**
+
+$$
+f(x \mid \nu, \mu, \sigma) =  \frac{\Gamma \left(\frac{\nu+1}{2} \right)} {\sqrt{\nu\pi}\Gamma \left(\frac{\nu}{2} \right)} \left(1+\frac{x^2}{\nu} \right)^{-\frac{\nu+1}{2}}
+$$
+
+where $\Gamma$ is the [gamma function](https://en.wikipedia.org/wiki/Gamma_function).
+
+**Cumulative Distribution Function (CDF):**
+
+$$
+F(y \mid \nu, \mu, \sigma) = 
+\begin{cases} 
+1 - \frac{1}{2} I_{\frac{\nu}{x^2 + \nu}} \left( \frac{\nu}{2}, \frac{1}{2} \right) & \text{for } x = \frac{y - \mu}{\sigma} \leq 0, \\[0.5em]
+\frac{1}{2} I_{\frac{\nu}{x^2 + \nu}} \left( \frac{\nu}{2}, \frac{1}{2} \right) & \text{for } x = \frac{y - \mu}{\sigma} > 0,
+\end{cases}
+$$
+
+where $I_x(a, b)$ denotes the [regularized incomplete beta function](https://en.wikipedia.org/wiki/Regularized_incomplete_beta_function).
+
+
+
+```{seealso}
+:class: seealso
+
+**Common Alternatives:**
+
+- [Normal Distribution](normal_distribution.md) - When $\nu \to \infty$, the t-distribution converges to the normal distribution.
+- [Cauchy Distribution](cauchy_distribution.md) - The Cauchy distribution is a special case of the Student's t-distribution with $\nu=1$.
+
+**Related Distributions:**
+
+- [Chi-Squared Distribution](chi_squared_distribution.md) - Student's t-distribution is a transformation of chi-squared distribution and can be obtained from chi-squared distribution and normal distribution.
```

Comment:
Not sure how useful this is in the context of preliz. Probably just ommit this section for this distribution